### ISSUE TITLE
Bug: Wrong responses being saved with defaultValue 

### ISSUE THREAD

**Issue by CarlosNZ** [openmsupply/application-manager-server#733]
See Angola product registration, Section 1, Page 2.

The queries for defaultValue are correct, and I've deleted the "dummy" application several times, but the responses for the pre-filled Applicant and Org info are going to the wrong questions
![Screen Shot 2022-04-27 at 5 13 26 PM](https://user-images.githubusercontent.com/5456533/165445637-bb88602d-6612-4ea2-b9c7-680fee820a3d.png)

Need to investigate further:
- does it do this on a real application, or just the "dummy" one?

**Comment by CarlosNZ** [comment]
Ah, I think I've realised why this is happening. It's related to this issue: https://github.com/openmsupply/application-manager-web-app/issues/1126

The reason why I kept seeing the erroneous result in the template builder was because my new template was still linked to an old config "dummy" application, and deleting the new one didn't make any difference. Or something like that.

Not sure exactly what's going on but this issue and that one are related and should be solved together.
